docs/example/README.md

```diff
@@ -1,24 +1,38 @@
-# OSM Example Usage Guide
+# OSM Demo Guide
```

Comment:
This is actually under `docs/examples` and seems to be an example guide. The demo guide is at `demo/README`. Should we rename this to `Manual Demo` guide?

Reply:
Yes we need to consolidate all demos into one source of truth on the landing page. We will ultimately remove this file. Once this get's merged I'll start working on updating the landing page with the content.;

Reply:
We have 2 demos, automated and manual. Both demo guides could co-exist. Did you have something else in mind?

Reply:
This will take over for the manual demo.

Reply:
Got it, thanks.